org.sofproject.gst.topo/src/org/sofproject/gst/topo/ui/wizards/GstNodeNewPage.java

```diff
@@ -78,7 +79,8 @@ public void createControl(Composite parent) {
 		data.grabExcessHorizontalSpace = true;
 		gstInspectToolCmd = new Text(inspectGroup, SWT.BORDER);
 		gstInspectToolCmd.setLayoutData(data);
-		gstInspectToolCmd.setText(gstNode.getGstInspectToolCmd());
+		gstInspectToolCmd.setText(Platform.getPreferencesService().getString("org.sofproject.gst.topo.ui.preferences",
```

Comment:
The qualifier name should be defined in IGstNodeConst (def based on GST_NODE_CORE_ID like others).